Spring-Project/src/main/java/com/epam/project/spring/dto/BucketDto.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.project.spring.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@Builder
+public class BucketDto {
```

Comment:
Hi Zenyk. It is a good practice to use one code style approach through entire project. If you use IntelliJ you can configure code style in the following way:

1. Please open Preferences -> Code Style -> Java -> Tab "Blank Lines".
2.  Specify the following values:
     **After class header**: 1
     **Around field**: 1
3. Save changes and please press press Ctrl - Alt - L to apply new code style rule.
Feel free to configure your own rules. Please use new formatting for all other classes. Thank you.

Reply:
Thanks. Resolved